templates/hooks/policy/sdk_read_one_post_set_output.go.tpl

```diff
@@ -1,5 +1,11 @@
-	if tags, err := rm.getTags(ctx, &resource{ko}); err != nil {
-		return nil, err
-	} else {
-		ko.Spec.Tags = tags
-	}
```

Comment:
Shouldn't we keep this hook? `GetPolicy` API Call doesn't return the list of tags

Reply:
It does actually :)

https://docs.aws.amazon.com/IAM/latest/APIReference/API_GetPolicy.html
https://docs.aws.amazon.com/IAM/latest/APIReference/API_Policy.html